changelog/_unreleased/2025-03-25-pin-promotions-for-admin-orders.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Pin promotions for admin orders
+author: Michel Bade
+author_email: m.bade@shopware.com
+author_github: @cyl3x
+---
+# Core
+* Added pinning promotion feature to `PromotionCollector`. Existing set of promotions will be kept and changed even if conditions are not matching anymore
+* Changed `PromotionCollector` to produce deleted notices for automatic promotion sole for the recalculation process
```

Comment:
This doesn't make sense.

Reply:
Does this make more sense?
* Changed `PromotionCollector` to add deletion notices for automatic promotions in the recalculation process